### PR TITLE
[SW-1295] Benchmarks: Configuration for External Backend

### PR DESCRIPTION
**Pull request by mn-mikke**

### CODE REVIEW

File: benchmarks/src/main/terraform/aws/modules/emr/variables.tf [hunk 1]
```diff
@@ -19,8 +19,11 @@ variable "aws_core_instance_count" {
...
 variable "aws_instance_type" {
-  default = "m5.xlarge"
+  default = "m5.2xlarge"
 }
 variable "sw_package_file" {
   default = "SUBST_PACKAGE_FILE"
 }
+variable "h2o_jar_file" {
+  default = "SUBST_H2O_JAR_FILE"
+}
```
Comment (jakubhava) [inline benchmarks/src/main/terraform/aws/modules/emr/variables.tf]: nit: nl

File: benchmarks/build.gradle [hunk 2]
```diff
@@ -37,16 +37,22 @@ shadowJar {
 
 task substituteTerraform() {
     doLast {
+        def outputStream = new ByteArrayOutputStream()
```
[the remark this answers is not in the document]
Comment (mn-mikke) [reply benchmarks/build.gradle]: Thanks! I've moved the logic to a separate task.

[the remark this answers is not in the document]
Comment (jakubhava) [reply benchmarks/build.gradle]: Thank you!

File: benchmarks/build.gradle [hunk 3]
```diff
@@ -34,14 +33,44 @@ shadowJar {
...
+}
+
+task buildH2OExtendedJarWithBenchmarks(type: Jar, dependsOn: buildH2OExtendedJar) {
+    manifest{
+        attributes 'Main-Class': 'water.hadoop.h2odriver'
+    }
+    version = ''
+    archiveBaseName = "h2o-extended-with-benchmarks"
+    zip64 true
+    from "$buildDir/classes/scala/main/"
```
Comment (jakubhava) [inline benchmarks/build.gradle]: 👍

**Pull request merged**